### ISSUE TITLE
Level 37 - us of 'else'

### ISSUE THREAD

**Issue by lauracumming** [ocadotechnology/codeforlife-admin#1084]
- [ ] update the solution for level 37 so it also uses 'else'

**Comment by rdcruzCFL** [comment]
@lauracumming  I can't do this. It does need doing though...

**Comment by faucomte97** [comment]
Hi @rdcruzCFL,

I am working on this and want to make sure we have a consistent approach to other levels.
I wrote a comment on [this other issue](https://github.com/ocadotechnology/rapid-router/issues/1799) relating to level 34 as I am unclear what we would like to do for that one.

Let me know 🙏🏻

**Comment by rdcruzCFL** [comment]
@faucomte97 Yes, in this case if, else if, else is the right approach. if, else if, else if seems strange to me, why not use else?
The relevant lesson is here: https://open-2c.gitbook.com/url/preview/site_YsOAP/teaching-resources/3.-selection/greater-than-greater-than-draft-complex-conditions-using-if-else-if-else

**Comment by lauracumming** [comment]
LGTM

**Comment by lauracumming** [comment]
Are all the solutions on Gitbook updated too @faucomte97 ?

**Comment by lauracumming** [comment]
@rdcruzCFL

**Comment by SKairinos** [comment]
LGTM

**Comment by faucomte97** [comment]
Updated the solutions in the teacher solution spaces (opened a change request for when we're ready), however the lesson plans are still referencing the benefit of having 3 separate if statements. What do we want to do about those? Reword them completely? Restructure them?

**Comment by lauracumming** [comment]
These are now replaced, I believe.

**Comment by faucomte97** [comment]
Not as far as I can tell. I can still see trailing else if blocks.
https://app.gitbook.com/o/-M_6U74XVytl1uT1MyQy/s/JNO9Ga5UyVUY3F0YDFPh/lower-ks3-resource-sheets/lks3-s4-4-resource-sheet
https://app.gitbook.com/o/-M_6U74XVytl1uT1MyQy/s/JNO9Ga5UyVUY3F0YDFPh/lower-ks3-resource-sheets/lks3-s5-1-resource-sheet

**Comment by lauracumming** [comment]
I thought else ifs were okay and it was just multiple ifs we didn't want?

**Comment by lauracumming** [comment]
Are these what we need?
[LKS3-S5-1-PDF.pdf](https://github.com/user-attachments/files/21792804/LKS3-S5-1-PDF.pdf)
[LKS3-S4-4_2025.pdf](https://github.com/user-attachments/files/21792805/LKS3-S4-4_2025.pdf)

**Comment by faucomte97** [comment]
Yes I believe so
